src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -52,6 +52,16 @@ CREATE FUNCTION find_shard_interval_index(bigint)
     RETURNS int
     AS 'citus'
     LANGUAGE C STRICT;
+-- remove tables from pg_dist_partition, if they don't exist i.e not found in pg_class
+delete from pg_dist_partition where not exists(select * from pg_class where pg_class.oid=pg_dist_partition.logicalrelid);
+select run_command_on_workers($$
```

Comment:
I think these tests might be flaky in the future.
can we do `SELECT 1 FROM run_command_...` so that the output never changes?